packages/paste-core/components/form/src/select/Select.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@styled-system/css';
+import {Box} from '@twilio-paste/box';
+import {ChevronDownIcon} from '@twilio-paste/icons/esm/ChevronDownIcon';
+import {Prefix} from '../shared/Prefix';
+import {Suffix} from '../shared/Suffix';
+import {FieldWrapper} from '../shared/FieldWrapper';
+import {safelySpreadFormControlProps} from '../shared/Utils';
+
+export interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  insertBefore?: React.ReactNode;
+  insertAfter?: React.ReactNode;
+  hasError?: boolean;
+  value: string | string[];
+}
+
+export interface SelectIconWrapperProps {
+  ref?: string;
+  insertAfter?: React.ReactNode;
+  children: React.ReactNode;
+}
+
+const SelectIconWrapper = React.forwardRef<HTMLSelectElement, SelectIconWrapperProps>(
+  ({children, insertAfter, ...props}, ref) => (
+    <Box
+      paddingTop="space10"
+      position="absolute"
+      display="inline-flex"
+      alignItems="center"
+      pointerEvents="none"
+      top="space120"
+      right={insertAfter ? 'space160' : 'space60'}
+      zIndex="zIndex10"
+      ref={ref}
+      {...props}
+    >
+      {children}
+    </Box>
+  )
+);
+
+const SelectElement = styled.select(
+  css({
+    appearance: 'none',
+    border: 'none',
+    background: 'transparent',
+    outline: 'none',
+    resize: 'none',
+    display: 'block',
+    width: '100%',
+    fontFamily: 'inherit',
+    fontSize: 'fontSize30',
+    lineHeight: 'lineHeight30',
+    fontWeight: 'fontWeightNormal',
+    color: 'colorText',
+    paddingTop: 'space30',
+    paddingRight: 'space40',
+    paddingBottom: 'space30',
+    paddingLeft: 'space40',
+    borderRadius: 'borderRadius20',
+    boxShadow: 'none',
+
+    '&:disabled': {
+      cursor: 'not-allowed',
+      color: 'colorTextWeak',
+    },
+  })
+);
+
+const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
+  ({disabled, hasError, insertBefore, insertAfter, children, size, multiple, ...props}, ref) => (
+    <FieldWrapper disabled={disabled} hasError={hasError}>
+      {insertBefore && <Prefix>{insertBefore}</Prefix>}
+      <Box display="flex" width="100%">
+        <SelectElement
+          aria-invalid={hasError}
+          ref={ref}
+          multiple={multiple}
+          size={multiple ? size : 0}
+          {...safelySpreadFormControlProps(props)}
+        >
+          {children}
+        </SelectElement>
+        {!multiple && (
+          <SelectIconWrapper insertAfter={insertAfter}>
+            <ChevronDownIcon decorative aria-hidden="true" />
```

Comment:
`decorative` should add `aria-hidden` for you

Reply:
Ah ok cool, I can remove this too.